runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessConfigureView/ProcessCredentials/ActionCredential.tsx

```diff
@@ -67,14 +79,10 @@ const ActionCredential: FunctionComponent<ActionCredentialProps> = ({
                         </ListItem>
                     </List>
                 </CredentialDetails>
-                <CredentialDelete>
+                <CredentialDelete $isPrimary={isPrimary}>
                     <DeleteOutlineIcon
-                        color={isPrimary ? 'disabled' : 'error'}
-                        sx={{[':hover']: { cursor: 'pointer' } }}
                         onClick={() => handleDeleteDialog(credentialId)}
-                        // Two lines to change after drag & drop (edit) functionality will be added
-                        // {...(!isPrimary && { onClick: () => handleDeleteDialog(credentialId) })}
-                        // sx={{...(!isPrimary && { [':hover']: { cursor: 'pointer' } })}}
+                        {...(!isPrimary && { onClick: () => handleDeleteDialog(credentialId) })}
```

Comment:
it it seems to me like you add `onClick` prop twice, or I don't get this right

Reply:
I think this is by accident (because in the past one line was commented as far as I remember